### PR TITLE
Fiveth requirement

### PR DESCRIPTION
**Pull request by dareddov**
In the admin panel there is a form, where an email, gender, age, points of interests can be specified for new and existing users. The form offer a possibility to dynamically specify the points of interest for the user, by adding additional fields to the form, without reloading the page. Each interest can have a name and a type (from the predefined collection of: ["health", "hobby", "work"]). When it comes to the number of interests, user can have any amount. New users created by admin have a "secret" password .

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -1,2 +1,29 @@
 class Admin::UsersController < Admin::BaseController
+  expose :user
+  expose :interests, from: :user
+  expose_decorated(:users, -> { User.all })
+
+  def create
+    user.password = 'secret'
```
Comment (Exelord) [inline app/controllers/admin/users_controller.rb]: Are you serious? :D Why??

**Review by Exelord — CHANGES_REQUESTED** [review]

**Comment by Exelord** [comment]
Please, add a description with the test instruction.

File: app/controllers/admin/users_controller.rb (hunk 1 above)
Comment (dareddov) [reply app/controllers/admin/users_controller.rb]: What do you mean? Should i use service for creating new user?

Comment (Exelord) [reply app/controllers/admin/users_controller.rb]: Why the password is always `secret`? You can use a random stirng generator or just allow user to specific one

Comment (dareddov) [reply app/controllers/admin/users_controller.rb]: "[...] New users created by admin should have a password set to "secret". [...]"

Comment (Exelord) [reply app/controllers/admin/users_controller.rb]: Great! Can you see now, what missing in this PR is? :)
Please, always add a PR's description to describe what exactly has been done. If the code and commits are not a self explonatory it's harde to get it from context without the informations and test scenario.

**Comment by dareddov** [comment]
@Exelord I updated PR, please see at this now, thanks. :)

**Pull request merged**